spec/controllers/admin/attachments_controller_spec.rb

```diff
@@ -82,19 +82,19 @@ module Alchemy
         end
 
         it "should set @while_assigning to true" do
-          get :new
+          alchemy_get :new
           expect(assigns(:while_assigning)).to eq(true)
         end
 
         it "should set @swap to params[:swap]" do
-          get :new, swap: 'true'
+          alchemy_get :new, swap: 'true'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.